src/components/Section/ProposalGovernanceSection.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+
+import Time from 'decentraland-gatsby/dist/utils/date/Time'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+
+import { ProposalAttributes, ProposalStatus } from '../../entities/Proposal/types'
+import { Vote } from '../../entities/Votes/types'
+import { ProposalPageState, SelectedChoice } from '../../pages/proposal'
+import { ChoiceProgressProps } from '../Status/ChoiceProgress'
+
+import ProposalVotingSection from './ProposalVoting/ProposalVotingSection'
+
+import './DetailsSection.css'
+import { ProposalPromotionSection } from './ProposalPromotionSection'
+import ProposalThresholdsSummary from './ProposalThresholdsSummary'
+
+type ProposalGovernanceSectionProps = Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> & {
```

Comment:
You can skip this `Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> &` as we are not using specific div attributes in this component